installer/templates/new/widgets/graph/graph.js

```diff
@@ -41,6 +46,12 @@ Widget.mount(class Graph extends Widget {
   currentValue() {
     return prettyNumber(prepend(this.current));
   }
+  getDefaultProps() {
```

Comment:
Ha right I added the correct version at the top, forgot to remove this one ofcourse. thnx